src/pages/staking-hub/dashboard/transactions.tsx

```
@@ -427,6 +435,31 @@ const PendingTransactionRow = ({ transaction }: { transaction: CustomSafeMultisi
     //cap: approve
     // data: { "method": "approve", "parameters": [ { "name": "spender", "type": "address", "value": "0x693Bac5ce61c720dDC68533991Ceb41199D8F8ae" }, { "name": "value", "type": "uint256", "value": "1000000000000000000000" } ] }
 
+    // *** Check for wrapper or transfer transactions
+    try {
+      if(
+        // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)